docs/_articles/en/apex/testing.md

```diff
@@ -3,27 +3,43 @@ title: Apex Tests
 
 ## Explore Your Apex Tests
 
-The Apex Tests sidebar provides several features. Here, you can see all your Apex tests at a glance. You can run one test method, the test methods in one class, or all your tests. You can view the results of your last test run. And you can jump from those results to the corresponding lines in your code. To access this sidebar, click the beaker icon (hover text: Test) in the view bar on the left side of the VS Code window. (If you don’t see this icon, make sure that the project you have open in VS Code contains an `sfdx-project.json` file in its root directory.)
+The Test view in the Side Bar provides several features such as LWC Tests and Apex Tests. To access the Test view, click the beaker icon ({% octicon beaker %}) in the Activity Bar on the left side of the VS Code editor. If you don’t see this icon, make sure that the project contains an `sfdx-project.json` file in its root directory. If the Test view is empty, check if the [Java setup](../getting-started/java-setup) is configured correctly.
 
-To run selected tests, in the Apex Tests view, hover over the name of a test method or class to reveal a play icon. Click the play icon (hover text: Run Single Test) to run a test method or all the methods in a class. To run all your tests, click the larger play icon at the top of the Apex Tests view (hover text: Run Tests).
+Use the Apex Tests feature to run one test method, test methods in one class, or all your Apex tests.
 
-After you run tests, the blue icons next to your classes and methods change to green icons (for passing tests) or red icons (for failing tests). To see details about your test runs, hover over the name of a test class in the sidebar.
+- Run Tests
 
-To jump to the definition of a test class, a test method that passed, or a method that you haven’t run yet, click its name in the sidebar. If you click the name of a failed test method, you jump to the assert statement that failed.
+  - Run a test for a single method: Hover over the name of a test method and click the play icon (hover text: Run Single Test).
+  - Run tests for all the methods in a class: Hover over the name of a test class and click the play icon (hover text: Run Tests).
+  - Run all tests: Hover over the Apex Tests view and click the play icon ({% octicon play %}).
 
-To clear your test results, click the refresh icon at the top of the sidebar (hover text: Refresh Tests).
+After you run Apex tests, SFDX: Re-Run Last Invoked Apex Test Class and SFDX: Re-Run Last Invoked Apex Test Method commands are available in the Command Palette.
 
-## Run Apex Tests from Within a File
+- View Test Results
+
+  - For passing tests, the blue icons next to the classes and methods change to green icons.
+  - For failing test, the icons turn red.
+
+To see the details of your test runs, hover over the name of a test class in the Side Bar.
+
+- Go to Test Class Definitions
```

Comment:
Is this supposed to be a header? I was noticing in the preview that it looked like the following were section headers as opposed to bullet points:  View Test Results, Go to Test Class Definitions, Clear Test Results, Refresh Apex Tests View

Reply:
I checked the preview and they appear as bullet points...

Reply:
Right, they are bullet points. I'm wondering if they should be headers? It looks odd to me that they are bullet points

Reply:
Yeah it seems like they should be headers maybe

Reply:
@hbc-bindu Any update on this piece? Should these be headers?

Reply:
Updated as headers.